src/Fixer/Whitespace/BlankLineBeforeControlStatementFixer.php

```diff
@@ -0,0 +1,264 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Whitespace;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerConfiguration\FixerOptionValidatorGenerator;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class BlankLineBeforeControlStatementFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
+{
+    /**
+     * @var array
+     */
+    private static $defaultConfiguration = [
+        'return',
+    ];
+
+    /**
+     * @var array
+     */
+    private static $tokenMap = [
+        'break' => T_BREAK,
+        'continue' => T_CONTINUE,
+        'declare' => T_DECLARE,
```

Comment:
I don't think there is such a conflict as `DeclareStrictTypesFixer` inserts the `declare` as first statement (as required by PHP). Since this fixer only insert a line if there is either a `;` or `{` before the statement it cannot be in conflict.